src/index.ts

```diff
@@ -36,10 +36,7 @@ function getDistPath() {
             console.log('Platform is not supported')
     }
 }
-const native_path = !process.env.FLUVIO_DEV
-    ? `${getDistPath()}/index.node`
-    : `${process.cwd()}/dist/${getDistPath()}/index.node`
-
+const native_path = `${process.cwd()}/dist/${getDistPath()}/index.node`
```

Comment:
This will be taken back to its previous state. I had some issues finding the actual binary when running tests locally.